src/docs/resources/faq.md

```diff
@@ -126,14 +126,14 @@ and address commonly-requested user features.
 
 Flutter includes:
 
-* Heavily optimized, mobile-first 2D rendering engine
+* Heavily optimized, the mobile-first 2D rendering engine
```

Comment:
Note the lack of periods on this list? That means that each entry need not be a full sentence. And, even with this change, it would still not be a grammatically correct sentence.